fineract-provider/src/main/java/org/apache/fineract/commands/domain/CommandSource.java

```diff
@@ -72,16 +71,24 @@ public class CommandSource extends AbstractPersistableCustom {
     @JoinColumn(name = "maker_id", nullable = false)
     private AppUser maker;
 
-    @Column(name = "made_on_date", nullable = false)
-    private LocalDateTime madeOnDate;
+    /*
+     * Deprecated: Columns and data left untouched to help migration.
+     *
+     * @Column(name = "made_on_date", nullable = false) private LocalDateTime madeOnDate;
+     *
+     * @Column(name = "checked_on_date", nullable = true) private LocalDateTime checkedOnDate;
+     */
+
+    @Column(name = "made_on_utc", nullable = false)
```

Comment:
Since in the native query you're referring to this column as madeOnDateUtc, do you think it would make sense to name it appropriately?
Instead of `made_on_utc` it could be `made_on_date_utc`.
Thoughts?

Reply:
You have eagle eyes. I was wondering change the native query instead. I started this kind of naming convention with all the new fields i have added, like: created_on_utc,
created_by,...

Would you mind if i stick to the column name but would change in the native query the alias?

Reply:
Fine for me, no worries. I just like things consistent, I'm not enforcing any type of naming ;-)

Reply:
Nah.. you are right... I just rename the column... :D